src/frontend/core/executor/AbstractExecutor.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+Copyright 2019 JasminGraph Team
```

Comment:
update copyright year and change `JasminGraph` to `JasmineGraph`

Reply:
Changed the file